### ISSUE TITLE
Projects: Conflict between user settings to process an item and package settings

### ISSUE THREAD

**Issue by cssm** [Trevoke/org-gtd.el#81]
I'm running into an issue with Projects.

In my config, I've added this line: 
(setq org-gtd-process-item-hooks '(org-priority org-set-effort org-set-tags-command))

Instead of the A priority symbol [#A] next to the task in the project:
[#A] Some project's headline > Task for that project

My output is:

In the agenda view:
<img width="325" alt="Screen Shot 2022-01-06 at 14 28 11" src="https://user-images.githubusercontent.com/32018264/148439967-1f250c70-185f-4ae7-8cb0-6f79fa0e868a.png">

In the file under the Projects headline:
<img width="217" alt="Screen Shot 2022-01-06 at 14 39 02" src="https://user-images.githubusercontent.com/32018264/148441271-54f19c75-e1ee-4cdd-ae72-ce5416919bb2.png">

I believe the package is adding [/] to tell me how many tasks are left however that is interfering with my priority settings. Can you assist me with this?

Thanks a ton for this package. It really keeps me organized and less stressed.

**Comment by Trevoke** [comment]
Confirmed bug, I'm on it, I know exactly how to fix this (writing the test, however, might be as annoying as ever :D )

I'm glad the package helps, that's the goal :D

**Comment by Trevoke** [comment]
This is fixed, it'll be available in MELPA on next build, and if you're using something else to pull straight from the repo, you should be able to pull the update.

You'll have to fix any broken projects yourself unfortunately :/

**Comment by cssm** [comment]
Thanks for the fix!

**Comment by cssm** [comment]
Hi again,

In my buffer, I get this error now:
<img width="641" alt="Screen Shot 2022-01-10 at 16 22 10" src="https://user-images.githubusercontent.com/32018264/148841400-cb7a7ff6-5371-4913-9111-fa57cf624811.png">

and 
 
The headline looks like this:
:[0/0]* [#A] A task

Can you help?

**Comment by Trevoke** [comment]
Hi @cssm ,

can you share your hooks variable and a sample inbox (was there one item in the inbox? maybe of them) that I can use to try to replicate this?

Thanks.

**Comment by cssm** [comment]
**Clarify buffer w/o processing:**
* Project One
:PROPERTIES:
:CATEGORY: inbox
:END:
:LOGBOOK:
- Captured [2022-01-10 Mon 16:55]
:END:
** Task 1
** Task 2
*** Subtask 1
*** Subtask 2
** Task 3
**After I attempt to let org-gtd store it with '<keybinding>'.**
:[0/0]* [#C] Project One
:PROPERTIES:
:CATEGORY: inbox
:Effort:   0:30
:END:
:LOGBOOK:
- Captured [2022-01-10 Mon 16:55]
:END:
** NEXT Task 1
** TODO Task 2
*** TODO Subtask 1
*** TODO Subtask 2
** TODO Task 3
I also get the error I screenshot above.

My hooks variable:
(setq org-gtd-process-item-hooks '(org-priority org-set-effort org-set-tags-command)

**Comment by Trevoke** [comment]
Well that's really strange, if there's `:` like that at the beginning, it indicates that it's part of an org drawer. 

Are you also using doom emacs, or spacemacs, or something like that?

I tried your example, set the hooks variable, captured your project exactly, in a brand-new org gtd setup, and when I processed the inbox and chose to make it a project, it worked and refiled to this, which is what I would have expected to happen

```
* Projects
  :PROPERTIES:
  :TRIGGER: next-sibling todo!(NEXT)
  :ORG_GTD: Projects
  :END:
*** [#C] Project One[0/3]                                           :sometag:
    :PROPERTIES:
    :CATEGORY: inbox
    :Effort:   0.5
    :END:
    :LOGBOOK:
    - Captured [2022-01-10 Mon 16:55]
    :END:
***** NEXT Task 1
***** TODO Task 2
******* TODO Subtask 1
******* TODO Subtask 2
***** TODO Task 3
```

**Comment by cssm** [comment]
I'm using my own config so it's possible there's another package doing something? I do have org-fancy-priorities installed and org's speed commands enabled. They could be doing something to headline? It's ok. I'll just manually fix things and file the project away. I appreciate your help!

**Comment by Trevoke** [comment]
Hmm..  Is your config somewhere where I could download it / test it out?

**Comment by cssm** [comment]
Not really and it's a mess. The problem stems from evil-commands.el. The keyboard shortcut I have set for C-e is evil-copy-from-below which explains the colon. It copies the first : of :PROPERTIES:.

**Comment by cssm** [comment]
Changing line 110 from (execute-kbd-macro (kbd "C-e")) to (end-of-line) fixes the problem and files the task away correctly.

**Comment by Trevoke** [comment]
This is fixed by #85 :D Thanks for the investigation and the PR!
